### PR TITLE
Events

### PR DESCRIPTION
**Pull request by nikovacevic**

### CODE REVIEW

**Review by AdamStack18 — APPROVED** [review]
LGTM

File: events.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Events
+
+Kubecost emits events when certain things happen. Those events are also recorded in an event log in the interest of diagnosing problems.
+
+## Event kinds
+
+* `AllocationSetSavedEvent` describes a saved AllocationSet
```
Comment (dwbrown2) [inline events.md:7]: did you consider going the next level of details on describing the verbs, e.g. save, load, aggregate? i like how simple this is, but when looking at save for example, i'd have question about what exactly this represents. is this save to disk? memory? would backup influence this?

**Review by dwbrown2 — COMMENTED** [review]
Looks great, just a couple questions inline! 

Also, would you plan to link this page from anywhere to make it discoverable by others?

File: events.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+# Events
```
Comment (srpomeroy) [inline events.md:1]: Titles need to be Setext H1 syntax I believe.
https://github.com/kubecost/docs/blob/main/markdown.md#titles

File: events.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+* `AssetSetReconciledEvent` described reconciling an AssetSet
+  * window: window of the set
+* `AssetSetComputeErrorEvent` described an error in computation
+* `AssetSetReconcileErrorEvent` described an error in reconciliation
+
+## Event log
+
+To access the most recent events in the event log, invoke `GET <kubecost>/model/etl/log`, which will provide the following, e.g.:
+
+```
```
Comment (srpomeroy) [inline events.md:42]: Though optional, I recommend adding language type to code block.
i.e.:
``` 
```json
```

**Review by srpomeroy — CHANGES_REQUESTED** [review]
Two minor tweaks (and one suggestion) based on our markdown guide.

**Comment by srpomeroy** [comment]
@nikovacevic Is this PR still relevant?
I made some tweaks to the formatting to better support our current and future docs platform.

**Comment by dwbrown2** [comment]
Know Niko's out this week... I believe the rationale for creating this still make sense, but I'm sure some things have changed given all the core work going on here. 

Let me know if I can do more to help with docs migration while he's OOO.

**Comment by srpomeroy** [comment]
Appreciate it @dwbrown2. We are trying to close out all non-release-related doc PRs by the end of next week.
It can wait until Niko is back unless you think someone else can answer quickly.

**Comment by dwbrown2** [comment]
@AjayTripathy are you familiar enough to say before Niko's back?

**Comment by nikovacevic** [comment]
I'm back! Just bad at following up on my own docs PRs, apparently. I'll gladly accept whatever suggested changes are here, and can merge after that.

File: events.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+* `AssetSetTotaledEvent` described totaling an AssetSet
+  * window: window of the set
+* `AssetSetReconciledEvent` described reconciling an AssetSet
+  * window: window of the set
+* `AssetSetComputeErrorEvent` described an error in computation
+* `AssetSetReconcileErrorEvent` described an error in reconciliation
+
+## Event log
+
+To access the most recent events in the event log, invoke `GET <kubecost>/model/etl/log`, which will provide the following, e.g.:
```
[the remark this answers is not in the document]
Comment (nikovacevic) [reply events.md:41]: No configurability. Just 200 today. We can add configurability later, I think!

```go
eventLog:   NewEventLog(200),
```

**Review by srpomeroy — APPROVED** [review]
LGTM!

**Pull request merged**